saleor/invoice/error_codes.py

```diff
@@ -0,0 +1,9 @@
+from enum import Enum
+
+
+class InvoiceErrorCode(Enum):
```

Comment:
We should add this enum to `SALEOR_ERROR_CODE_ENUMS` in `saleor.graphql.core.utils`.

Reply:
It was added already.